src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -2251,7 +2251,7 @@ mobile_apis::Result::eType MessageHelper::VerifyImageFiles(
   return mobile_apis::Result::SUCCESS;
 }
 
-mobile_apis::Result::eType MessageHelper::VerifyImage(
+mobile_apis::Result::eType MessageHelper::VerifyImageApplyPath(
```

Comment:
@tkireva Please modify the previous Unit test for `VerifyImage` to be suitable for the `VerifyImageApplyPath` method. Also add a new test for the `VerifyImage`.